hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/debug/TestLDBCli.java

```diff
@@ -341,6 +341,43 @@ void testScanOfPipelinesWhenNoData() throws IOException {
     assertEquals("", stderr.toString());
   }
 
+  @Test
+  void testScanWithRecordsPerFile() throws IOException {
+    // Prepare dummy table
+    prepareTable(KEY_TABLE, false);
+
+    String scanDir1 = tempDir.getAbsolutePath() + "/scandir1";
+    // Prepare scan args
+    List<String> completeScanArgs1 = new ArrayList<>(Arrays.asList(
+        "--db", dbStore.getDbLocation().getAbsolutePath(),
+        "scan",
+        "--column-family", KEY_TABLE, "--out", scanDir1,
+        "--max-records-per-file", "2"));
+
+    File tmpDir1 = new File(scanDir1);
+    tmpDir1.deleteOnExit();
+
+    int exitCode1 = cmd.execute(completeScanArgs1.toArray(new String[0]));
+    assertEquals(0, exitCode1);
+    assertTrue(tmpDir1.isDirectory());
+    assertEquals(3, tmpDir1.listFiles().length);
```

Comment:
Can use `records_count / max_records_per_file # need Math.ceil` replace this hard code, we can add a parameter  records_count  for method `prepareTable`.